sdk/storage/storage-blob/swagger/README.md

```diff
@@ -12,7 +12,7 @@ enable-xml: true
 generate-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../src/generated
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/8c1f9b739165035d9ced32761cebb19125bce233/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-06-12/blob.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/b50bcdde18465bbf04937a9ee73607753d24d65c/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-10-02/blob.json
```

Comment:
Please replace:
```
use-extension:
  "@autorest/typescript": "6.0.0-dev.20210218.1"
```
with
```
typescript: true
```

Reply:
With option "typescript", it will try to use generator with version of "5.**" by default. Our feature would requires "6.**" generators.

Reply:
Just using the flag tells autorest to get the latest version which should be a v6. Would you mind running `autorest --reset` first and then try again?

Reply:
Thanks a lot for the comments. I tried as your suggested, it regenerates codes with @autorest/typescript' (latest->6.0.0-beta.6). Seems the generated code has many changes which are not related to the feature. I'll have another PR for the generator.

Reply:
Opened an issue to track it: https://github.com/Azure/azure-sdk-for-js/issues/16362

Reply:
Ah right, because it will use core v2 by default. You could add `use-core-v2: false` to your swagger readme to disable that for now.